src/utils.js

```diff
@@ -0,0 +1,23 @@
+import moment from 'moment';
+
+export function isSameDay(currentMessage = {}, diffMessage = {}) {
```

Comment:
You can take the opportunity and also refactor this using the [`isSame` moment's function](http://momentjs.com/docs/#/query/is-same/).

Something like 

```js
return moment(diffMessage.createdAt).isSame(currentMessage.createdAt)
```